data/drak/drak missions.txt

```diff
@@ -30,7 +35,12 @@ mission "Drak guarding Korath space"
 mission "Drak guarding Sayaiban"
 	invisible
 	landing
-	npc kill
+	repeat
+	to offer
+		not "Drak guarding Sayaiban: active"
+	to complete
+		never
```

Comment:
Since this mission has an `on enter` conversation, the conversation will repeat each time the Archon respawns. The conversation should probably be broken off into its own mission.